src/Uno.UI/UI/Xaml/Controls/Popup/PlacementPopupPanel.cs

```diff
@@ -60,8 +60,13 @@ internal abstract partial class PlacementPopupPanel : PopupPanel
 
 		protected PlacementPopupPanel(Popup popup) : base(popup)
 		{
+			Loaded += (s, e) => Windows.UI.Xaml.Window.Current.SizeChanged += Current_SizeChanged;
+			Unloaded += (s, e) => Windows.UI.Xaml.Window.Current.SizeChanged -= Current_SizeChanged;
```

Comment:
So that's the problem? The size wasn't reevaluated?

Reply:
That fixes the issue where the location is not updated properly (or updated +1) when the window size changes.